.github/workflows/lock-threads.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock Closed Threads'
+
+on:
+  schedule:
+    - cron: '0 7 * * *'  # 7am UTC, daily
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```

Comment:
Should we pin third party actions on git sha, for security reasons ?

Reply:
I don't see any problem on doing this.

Reply:
We don't do that for any of our existing usage.

It might make sense to do that across the board separately.